pennylane/labs/tests/resource_estimation/ops/qubit/test_parametric_ops_single_qubit.py

```diff
@@ -36,26 +36,62 @@ def test_rotation_resources(epsilon):
     assert gate_types == _rotation_resources(epsilon=epsilon)
 
 
-class TestRZ:
-    """Test ResourceRZ"""
+class TestPauliRotation:
+    """Test ResourceRX, ResourceRY, and ResourceRZ"""
 
-    @pytest.mark.parametrize("epsilon", [10e-3, 10e-4, 10e-5])
-    def test_resources(self, epsilon):
+    params = list(zip([re.ResourceRX, re.ResourceRY, re.ResourceRZ], [10e-3, 10e-4, 10e-5]))
+
+    @pytest.mark.parametrize("resource_class, epsilon", params)
```

Comment:
I think what we want to do is keep testing RZ with all 3 epsilon values, but then also to test RX, RY with this. It should be easy to do by nesting the parametrize calls: 

```suggestion
    r_classes = [re.ResourceRX, re.ResourceRY, re.ResourceRZ]
    epsilon_values = [10e-3, 10e-4, 10e-5]

    @pytest.mark.parametrize("epsilon", epsilon_values)
    @pytest.mark.parametrize("resource_class", r_classes)
```